app/components/primer/image_crop.html.erb

```diff
@@ -0,0 +1,14 @@
+<%= render Primer::BaseComponent.new(**@system_arguments) do %>
+  <% if loading.present? %>
+    <%= loading %>
+  <% else %>
+    <div data-loading-slot>
+      <%= render(Primer::SpinnerComponent.new(size: :large, classes: "flex-1")) %>
+    </div>
+  <% end %>
+
+  <input type="hidden" data-image-crop-input="x" name="cropped_x">
+  <input type="hidden" data-image-crop-input="y" name="cropped_y">
+  <input type="hidden" data-image-crop-input="width" name="cropped_width">
+  <input type="hidden" data-image-crop-input="height" name="cropped_height">
```

Comment:
are those inputs always needed? From the docs, it looks like they are optional.
Maybe we could add a flag to the component to add them?

Reply:
These inputs are always needed for the GitHub monolith. We send the coordinates and dimensions to the server with the original image, and the server does the actual cropping. I was thinking about making it optional or overwritable but decided on keeping it in to keep the component simple in the initial ship.

Open to suggestions on how we can make this better.

Reply:
Perhaps we take a to-do to document how this component passes its data to the server?